README.md

```diff
@@ -19,6 +26,48 @@ By providing an OCI artifact definition, the community can continue to innovate,
 
+## Overview of Registry Content Delivery
+
+Registry content delivery can be broken down into 3 distinct components:
+
+| | |
+|-|:-:|
+| [Registry](#registry) |<img src=./media/registry-layer.png height=100> |
+| [Manifest Schemas](#manifest-schemas) |<img src=./media/manifest-layer.png height=40> |
+| [Artifacts](#artifacts) |<img src=./media/artifact-layer.png height=100> |
+
+### Registry
+
+A registry, as an implementation of the [OCI Distribution Spec][distribution-spec], serves a few primary roles:
+
+* Persistence of content, stored as blobs
+* Aggregation of blobs, defined by manifests
+* A REST endpoint for content discovery
+* Content delivery
+* Authentication & authorization
+
+### Manifest Schemas
+
+For a registry to store collections of content, it must have well known schemas to uniquely describe each content addressable object. The [OCI Manifest][image-manifest] and [OCI Index][image-index] are two well known schemas that implementations of the [OCI Distribution Spec][distribution-spec] MUST support.
+
+While it's possible for registries to implement additional manifest schemas, the authors of the OCI manifest and index had the forethought of flexibility, enabling the majority of scenarios we know today.
+
+### Artifacts
+
+Artifacts, like docker and OCI images are well known types of artifacts. Using the same schemas and persistence, additional artifacts can be defined, persisted and served with OCI distribution spec compliant registries.
+
+OCI Artifacts generalizes registries from uniquely storing OCI Images, to generic means storing additional artifact types.
+
+## Defining OCI Artifact Types
+
+[Registries][def-registry], vulnerability scanners and artifact tooling must understand the types of artifacts they support. Registry scanning tools may only support a subset of artifact types, or they may need to apply different scanning methods based on the artifact type.
+
+If a security scanning solution were to scan all types, it would fail when it encounters unsupported types, representing false negatives. By differentiating types, a registry scanning solution can ignore unknown types, representing a known state. As new artifact types become [well known][def-well-known-types], scanners can expand the types they offer, providing a more complete known state.
+
+Artifact tooling must also know the types they support. The docker and containerD client know how to instance container images. However, they are not intended to instance Helm Charts or Singularity images. By defining the artifact type, registries can present the type to their users, and tools pulling artifacts from a registry can determine if they can support the specific type before encountering a runtime error.
+
+Artifacts are defined by setting the `manifest.config.mediaType` to a globally unique value. The `config.mediaType` of `application/vnd.oci.image.config.v1+json` is reserved for artifacts intended to be instanced by docker and [containerD][containerd].
```

Comment:
yes but to much detail for the readme...we should cover this in the spec vs readme..

s/containerD/containerd/